gs-product-configurator/gspc-add-order-meta-to-gf-entry.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Gravity Shop // GS Product Configurator // Add order metadata to GF entries on export.
+ *
+ * Instructions:
+ *   1. Install per https://gravitywiz.com/documentation/how-do-i-install-a-snippet/
+ */
+
+if ( is_callable( 'gs_product_configurator' ) ) :
+
+	function gspc_add_order_meta_to_gf_entry_metas() {
+		/**
+		 * IMPORTANT!!! Update this with the WC metadata keys you want to add.
+		 * You may optionally include a label to use during e.g. export like so:
+		 *     '_billing_phone' => 'Phone',
+		 * If a label is not included, labels are generated thus:
+		 *     e.g. 'Billing Phone' is used for '_billing_phone'.
+		 *
+		 * Example: return array(
+		 *  '_order_key',
+		 *  '_billing_first_name',
+		 *  '_billing_last_name',
+		 *  '_billing_email',
+		 *  '_billing_phone' => 'Phone',
+		 *  '_shipping_first_name',
+		 *  '_shipping_last_name',
+		 *  '_cart_discount',
+		 *  '_created_via',
+		 * );
+		 *
+		 * Please see class WC_Order_Data_Store_CPT in the WC plugin.
+		 * You may also check your DB records or the extension files
+		 * to see other order metadata keys that your installed WC
+		 * extensions might be storing.
+		 */
+		return array();
```

Comment:
Here's a suggestion of moving the example into the returned keys. I think it's safe to put a few defaults here so it will do things simply by adding the snippet.

```suggestion
		 *
		 * Please see class WC_Order_Data_Store_CPT in the WC plugin.
		 * You may also check your DB records or the extension files
		 * to see other order metadata keys that your installed WC
		 * extensions might be storing.
		 */
		return array(
		    '_order_key',
		    '_billing_first_name',
		    '_billing_last_name',
		    '_billing_email',
		    '_billing_phone' => 'Phone',
		    '_shipping_first_name',
		    '_shipping_last_name',
		    '_cart_discount',
		    '_created_via',
		 * );
```